GcmEndpoints/README.md

```diff
@@ -106,136 +96,38 @@ class GcmRegistrationAsyncTask extends AsyncTask<Void, Void, String> {
                         abstractGoogleClientRequest.setDisableGZipContent(true);
                     }
                 });
-                // end of optional local run code
-
-            regService = builder.build();
-        }
-
-        String msg = "";
+        Registration regService = builder.build();
         try {
-            if (gcm == null) {
-                gcm = GoogleCloudMessaging.getInstance(context);
-            }
-            String regId = gcm.register(SENDER_ID);
-            msg = "Device registered, registration ID=" + regId;
-
-            // You should send the registration ID to your server over HTTP,
-            // so it can use GCM/HTTP or CCS to send messages to your app.
-            // The request to your server should be authenticated if your app
-            // is using accounts.
-            regService.register(regId).execute();
-
+            regService.register(token).execute();
         } catch (IOException ex) {
             ex.printStackTrace();
```

Comment:
maybe add Log.e("some-gcm-registration-tag", "Failed to register with server", ex)
or even allow this exception to be handled by the parent method which will notify the user better?